src/modules/event/view/event-edit-item-view.js

```diff
@@ -198,6 +230,22 @@ export default class EventEditItemView extends AbstractStatefulView {
     if (this._state.typeItem.offers.length !== 0) {
       this.element.querySelector('.event__available-offers').addEventListener('change', this.#availableOfferChangeHandler);
     }
+
+    this.#setDatePicker('dateFrom', `#event-start-time-${this._state.id}`);
+    this.#setDatePicker('dateTo', `#event-end-time-${this._state.id}`);
+  }
+
+  #setDatePicker(picker, selector) {
+    this.#datePickers[picker] = flatpickr(
+      this.element.querySelector(selector),
+      {
+        enableTime: true,
+        // eslint-disable-next-line camelcase
+        time_24hr: true,
```

Comment:
```suggestion
        'time_24hr': true,
```
а дизейбл линта надо удалить